src/lib/Form/Type/ChoiceList/Loader/ContentEditTranslationChoiceLoader.php

```diff
@@ -50,14 +54,16 @@ public function __construct(
         ?ContentInfo $contentInfo,
         LookupLimitationsTransformer $lookupLimitationsTransformer,
         array $languageCodes,
-        LocationService $locationService
+        LocationService $locationService,
+        ?Location $location
```

Comment:
Doesn't it make sense to always require this parameter and drop the `LocationService` dependency as a result?

Reply:
Aside from BC, I think this would be a good idea.

Reply:
@Steveb-p 
when it comes to question `1` I think we can skip reloading the `Location`, the same `Location` is still checked when accessing this content before. 
`2`: Mandatory with `null` as a default or without `null`? Based on the parts in system where this service is called, we cannot guarantee we will have `Location` instead of `null`. If that's a `null` we won't really be able to load limitations for the given content, that's why we could help ourselves with `mainLocationId` which in fact should be loaded.